lib/src/main/java/edu/utexas/tacc/tapis/files/lib/clients/SSHDataClient.java

```diff
@@ -446,8 +446,8 @@ public void delete(@NotNull String path) throws IOException, NotFoundException,
   {
     // Get path relative to system rootDir and protect against ../..
     String relativePathStr = PathUtils.getRelativePath(path).toString();
```

Comment:
Can we put SshSessionContext into sessionholder to reduce contention of lock on release of session